app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,44 @@
+footer {
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  margin: 0 auto;
+  height: 200px;
+  border-radius: 15px;
```

Comment:
Can you use lightgrey for the background-color and border-radius 20px instead ? That's to fit in the the existing design